### DISCUSSION TITLE
AttributeError: 'NoneType' object has no attribute 'send'

### DISCUSSION THREAD

**Discussion by during-code** [oracle/python-oracledb#427] [Q&A]
Hello, may I ask the result of my connection calling connection.is_healthy() is TRUE, but why does the query or insert operation get an error AttributeError: 'NoneType' object has no attribute 'send'

**Comment by anthony-tuininga** [comment]
Can you supply an example that demonstrates the problem? You should not get that error to begin with, and if you do, you should see `is_healthy()` return `False.
